bundles/best_practices.rst

```diff
@@ -166,6 +166,72 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of Symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+            - $HOME/symfony-bridge/.phpunit
+
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported Symfony version with the latest PHP version
+            - env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
```

Comment:
`--prefer-stable` is not the default. But as we recommend to keep the default `stable` minimum-stability in composer.json (and only have 1 other job changing it to test latest dev), it would make no difference here in most cases (as preferring stable over unstable has no effect when unstable is entirely forbidden anyway)